.idea/compiler.xml

```diff
@@ -6,10 +6,12 @@
         <sourceOutputDir name="target/generated-sources/annotations" />
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
         <outputRelativeToContentRoot value="true" />
+        <module name="networking-assistant" />
```

Comment:
I believe this is the result of you naming your modules differently. I'm keeping the Pascal case one.

Reply:
Fixed